srcpkgs/ngx_headers_more/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'ngx_headers_more'
+pkgname=ngx_headers_more
+version=0.33
+revision=1
+_nginx_version=1.18.0
```

Comment:
Could you add a note to the nginx package near the version field that nginx version needs to be kept in sync with all its modules?

Reply:
Or maybe we should make this a subpackage of nginx? What do you think?

Reply:
And we would require updating rebuilds of everything, including rebuilds of nginx if one module updates.

Reply:
> And we would require rebuilds of all modules, including rebuilds and updates of nginx if one module gets updated.
> So its good that they are split up into separate templates.

Ok, let's just add a note to the nginx package.